tests/test_state_props/test_is_separable.py

```diff
@@ -53,10 +53,10 @@ def test_ppt_low_rank():
     u, s, v_h = np.linalg.svd(rho)
     rho_cut = u[:, :m-1] @ np.diag(s[:m-1]) @ v_h[:m-1]
     rho_cut = rho_cut / np.trace(rho_cut)
-    pt_state_alice = partial_trace(rho_cut, 2, dim=[3, 2])
+    pt_state_alice = partial_trace(rho_cut, [1], dim=[3, 2])
```

Comment:
This line yields the following warning:

```
tests/test_state_props/test_is_separable.py::test_ppt_low_rank
  /Users/vincent.russo/Projects/research/toqito/tests/test_state_props/test_is_separable.py:56: DeprecationWarning: Argument 'dim' to partial_trace is deprecated: Use 'dimensions' instead.
    pt_state_alice = partial_trace(rho_cut, [1], dim=[3, 2])

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
```

Basically, we just need to alter the `dim=` part to `dimensions=`.